pkg/dpll/dpll.go

```diff
@@ -585,7 +614,6 @@ func (d *DpllConfig) stateDecision() {
 			d.sourceLost = true
 		}
 		d.state = event.PTP_FREERUN
-		d.phaseOffset = FaultyPhaseOffset
```

Comment:
I removed all the phase offset mocking. I think it's no longer needed

Reply:
d.phaseOffset is now initialized in pin notification handler before calling this function. The offset is not updated when device notification is handled (it is not conveyed by the notification). I think in general it should be passed, but transients are possible. For example, when we get devices, phase offset is not yet initialized. But it will be in less than a second.